std_msgs/Header header

float64 latitude 	# target latitude [deg]
float64 longitude 	# target longitude [deg]
float32 altitude 	# target altitude [m] (relative to sea level)
float32 north_speed 	# target north speed [m/s]
float32 east_speed 	# target east speed [m/s]
float32 down_speed 	# target down speed [m/s]
